www/js/survey/enketo/enketo-notes-list.js

```diff
@@ -32,7 +32,7 @@ angular.module('emission.survey.enketo.notes-list', [])
     $scope.deleteEntry = (index, entry) => {
       console.log("Deleting entry", entry);
 
-      const dataKey = entry.metadata.key;
+      const dataKey = entry.key || entry.metadata.key;
```

Comment:
do we actually have to have the `dataKey` as a field for the notes list?
We already support different surveys and different labels.
And the data is going to come back as `trip-additions` (please see server code)
So it seems like we don't really need the data key to be configurable.

Keeping a single data key for "one per trip" and one for "n per trip" objects respectively can also make it easier for the downstream code, such as the public dashboard.

Again, check the server code for an example

Reply:
I thought the key was necessary here because for deletion, we need to put an entry with status DELETED to the UserCache. To my understanding we are using a different key for trip- and place- level additions (`manual/trip_addition_input` and `manual/place_addition_input`)
so notes-list needs to know which key to use when it adds the DELETED entry.